contracts/TellorStake.sol

```diff
@@ -301,19 +301,17 @@ contract TellorStake is TellorTransfer {
 
             //Reduce the staker count
             uints[_STAKE_COUNT] -= 1;
-
-            //Update the minimum dispute fee that is based on the number of
-            // Not ideal, but allows to keep updateMinDosputeFee in the extension contract
-            addresses[_EXTENSION].delegatecall(
```

Comment:
it doesn't really add much and I wanted to avoid another delegate call.  the update dispute fee is open, so anyone can run it.  But again, I was back and forth